design/0008-community-plugins.md

```diff
@@ -0,0 +1,48 @@
+# Community Plugins
+
+```
+Author: Yudai Kiyofuji <yudai.kiyofuji@emqx.io>
+Status: Draft
+First Created: 2021-02-21
+EMQ X Version: 4.3
+```
+
+## Abstract
+
+This proposal suggests ways to encourage plugin development by community. 
+
+## Motivation
+
+Allowing people to develop their own plugins is a good way for EMQX to gain popularity.
+To achieve this, it is nice to `approve` some community plugins and let people use them.
+
+## Design
+
+In rebar.config.erl, we can add a list of approved community plugins.
+
+```erlang
+community_deps() ->
```

Comment:
I'd propose to have this in a separate file e.g. `community-plugins`
and consult it from `rebar.config.erl`

Reply:
the advantage of having it in a separate file is to keep minimum lines of such in `rebar.config.erl` which may otherwise cause more lines of conflicts when porting changes to enterprise.

Reply:
updated the doc